### ISSUE TITLE
Client configurations outdated(vue.config.js)

### ISSUE THREAD

**Issue by robbykrlos** [laravel-enso/enso#372]
This is  a **bug?**.

### Prerequisites
* [x] Are you running the latest version?
* [x] Are you reporting to the correct repository?
* [x] Did you check the documentation?
* [x] Did you perform a cursory search?

### Description

While investigating this #352 - I have checked some files from client side - just to be sure we're in sync.

Not sure if a bug or not, but shouldn't the laravel-enso/enso configuration released in 4.6.0 reflect the latest configuration changes (with all required changes from all upgrade steps 4.1 ... 4.4,  4.5, 4.6? 

My logic says that a new enso installation will not work, due to @ui / @core alias missmatches. But maybe I'm wrong.

Latest package (one example file)
https://github.com/laravel-enso/enso/blob/6e6f14c5e1664765ca05d186826c69d1fac5fc0b/client/vue.config.js#L40

Release notes from 4.4.0:
> the @core alias has been renamed to @ui, so update all usages, including
> - local resources (pages, components)
> - the client/.eslintrc.js file
> ...

PS: I've found one file, but could be more, didn't check all configs.

**Comment by raftx24** [comment]
Hi @robbykrlos.
Yes, you are right, even though we don't use @ui here but anyway we should use @ui in `.eslintrc.js` too.
could you please create a PR for it?
thank you

**Comment by robbykrlos** [comment]
Sure. Will do.

**Comment by robbykrlos** [comment]
> Yes, you are right, even though we don't use @ui here but anyway we should use @ui in `.eslintrc.js` too.

Hi again, 

So now that I'm actually doing this, I further more confused by your last comment:

So, my initial ticket was related to `vue.config.js` - which had a bad outdated alias - which I fixed by PR #373

`.eslintrc.js` has no problem from what I see, the alias is correct in this file:
```js
alias: {
                map: [
                    ['@ui', `${__dirname}/node_modules/@enso-ui/ui/src`],
                    ['@root', `${__dirname}/src/js`],
```

Probably typo and you meant vue.config.js?
